execution-engine/src/fs.rs

```diff
@@ -662,19 +1024,53 @@ impl FileSystem {
         Ok(rst)
     }
 
-    /// The stub implementation of `path_create_directory`. Return unsupported error `NoSys`.
+    /// The implementation of `path_create_directory`.
     #[inline]
     pub(crate) fn path_create_directory<T: AsRef<Path>>(
         &mut self,
         fd: Fd,
-        _path: T,
+        path: T,
     ) -> FileSystemResult<()> {
+        info!(
```

Comment:
Are these logging messages just development aids, or will they stay in the code?

Reply:
It will be erased before merging.